src/components/Book.css

```diff
@@ -12,14 +12,12 @@
 
 .book-holder {
   text-align: left;
-  margin: auto 0 auto 40px;
 }
```

Comment:
- [ ] As requested by the previous code reviewer, which is a requirement for the project, please make sure your implementation matches the [design ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png)